.gitignore

```diff
@@ -1,3 +1,7 @@
+###> phpunit/phpunit ###
```

Comment:
Is the `### ... ###` wrapping here also written by Flex? Why is it interested in PHPUnit rather than Symfony packages?

Reply:
Yep. It can work with any vendor.